src/site/content/en/blog/use-thumbor/index.md

```diff
@@ -0,0 +1,211 @@
+
+<figure class="w-figure  w-figure--center">
+  <img src="./Use-thumbor8.jpg" alt="Image that is 500 pixels tall" class="w-screenshot">
+  <figcaption>Image resized to a height of 500 pixels<figcaption>
+</figure>
+
+
+
+You can also resize images to a percentage of the original by using the [proportion](https://thumbor.readthedocs.io/en/latest/proportion.html) filter. If size is specified in conjunction with the proportion filter, the image will be resized, and then the proportion filter will be applied.
+
+Try it now:
+
+1. Resize the image to 50% of the original: <a href="http://34.67.235.246:8888/unsafe/filters:proportion(.5)/https://web.dev/backdrop-filter/hero.jpg" target="_blank" rel="noreferrer">http://34.67.235.246:8888/unsafe/filters:proportion(.5)/https://web.dev/backdrop-filter/hero.jpg</a>
+
+<figure class="w-figure  w-figure--center">
+  <img src="./Use-thumbor9.jpg" alt="Image that is 50% the size of the original" class="w-screenshot">
+  <figcaption>Image resized to 50% the size of the original</figcaption>
+</figure>
+
+
+2. Resize the image to a width of 1000 pixels, then resize the image to 10% of its current size: <a href="http://34.67.235.246:8888/unsafe/1000x/filters:proportion(.1)/https://web.dev/backdrop-filter/hero.jpg" target="_blank" rel="noreferrer">http://34.67.235.246:8888/unsafe/1000x/filters:proportion(.1)/https://web.dev/backdrop-filter/hero.jpg</a>
+
+<figure class="w-figure  w-figure--center">
+  <img src="./Use-thumbor10.jpg" alt="Image that is 100 pixels wide" class="w-screenshot">
+  <figcaption>Image resized to a width of 100 pixels</figcaption>
+</figure>
+
+
+These methods are just a few of Thumbor's many cropping and resizing options. To read about other options, check out [Usage](https://github.com/thumbor/thumbor/wiki/Usage).
+
+
+### File Formats
+
+The [format](https://thumbor.readthedocs.io/en/latest/format.html) filter converts images to `jpeg`, `webp`, `gif`, or `png`. Keep in mind that if you're optimizing for performance you should [use either JPEG or WebP](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) as PNG and GIF files tend to be significantly larger and do not compress as well.
+
+Try it now:
+
+1. Convert the image to WebP. If you open the **Network** panel of DevTools the document's **Content-Type response header** shows that the server returned a WebP image: <a href="http://34.67.235.246:8888/unsafe/filters:format(webp)/https://web.dev/backdrop-filter/hero.jpg" target="_blank" rel="noreferrer">http://34.67.235.246:8888/unsafe/filters:format(webp)/https://web.dev/backdrop-filter/hero.jpg</a>
+
+<figure class="w-figure  w-figure--center">
+  <img src="./Use-thumbor11.jpg" alt="DevTools screenshot showing the content-type (WebP) of an image" class="w-screenshot">
```

Comment:
This image looks low-quality but I can fix it in a subsequent PR

cc @robdodson

Reply:
I updated this image. It's still not super high quality bc I was trying to keep the file size reasonable but it's definitely better than the previous version.